package.json

```diff
@@ -37,11 +36,11 @@
     "url": "https://github.com/canjs/can-validate-validatejs/issues"
   },
   "dependencies": {
-    "can-validate": "^1.0.0-pre.1",
-    "validate.js": "^0.11.1"
+    "can-util": "3.2.2",
+    "validate.js": "0.11.1"
```

Comment:
You might want to do the same here ... this will allow people to use other versions of validate.js if they want to.